### ISSUE TITLE
Feature Request: Automatically change preview theme following system dark/light setting

### ISSUE THREAD

**Issue by Yue-Zhengyuan** [shd101wyy/vscode-markdown-preview-enhanced#603]
I would like to see that the extension can automatically change the preview theme based on the system dark/light setting. It can be done in a way similar to how VS Code itself changes the theme: the user can select which theme to be used in light/dark mode, and whether the automatic change is enabled.

**Comment by noonuid** [comment]
Yes, it is a very useful feature.

**Comment by mert-kurttutan** [comment]
I agree. Having the contrast between markdown and my vscode is suboptimal when I use dark mode on vscode.
Is there any progress on this feature?

**Comment by kenberkeley** [comment]
For now, what we can do is right-click and set the theme manually ⬇️

![image](https://user-images.githubusercontent.com/13287948/210288545-0fb9b296-625d-42b8-8dc8-c99d6c651c24.png)
